### PR TITLE
Update method middlewares to support permission checking

### PR DESCRIPTION
**Pull request by justinvdm**

### CODE REVIEW

**Comment by justinvdm** [comment]
:+1:

File: src/routes/user.js [hunk 1]
```diff
@@ -4,7 +4,5 @@ const { read } = require('src/middleware/api/methods');
 
 
 module.exports = [
-  _.get(
-    '/user/permissions/',
-    read(user.getPermissions))
+  _.get('/user/permissions/', read(user.getPermissions))
```
Comment (justinvdm) [inline src/routes/user.js:7]: This route needs to change, planning on doing that outside of this PR.

**Comment by justinvdm** [comment]
@rudigiesler ready for review

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**